index.js

```diff
@@ -157,6 +157,8 @@ function getType(value) {
 
 function encodingLength(value) {
   var type = getType(value)
+  if ('function' !== typeof encodingLengthers[type])
+    throw new Error('unknown type:' + type + ', ' + JSON.stringify(value))
```

Comment:
This check is important before we attempt to call `encodingLengthers[type]` on the line below.